ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -187,6 +187,19 @@ switch:
                           switch.default_evpn_role, 'none') %}
   evpn_role: {{ switch_evpn_role }}
 
+{# switch.evpn_gateway #}
+{%     set switch_evpn_gateway = switch_data.combined.evpn_gateway.evpn_l2.enabled is arista.avd.defined(true) or
+                                 switch_data.combined.evpn_gateway.evpn_l3.enabled is arista.avd.defined(true) or
+                                 switch_data.combined.evpn_gateway.vpn_ipv4.enabled is arista.avd.defined(true) %}
+  evpn_gateway: {{ switch_evpn_gateway | arista.avd.default(false) }}
+  evpn_gateway_evpn_l2: {{ switch_data.combined.evpn_gateway.evpn_l2.enabled | arista.avd.default(false) }}
+  evpn_gateway_evpn_l3: {{ switch_data.combined.evpn_gateway.evpn_l3.enabled | arista.avd.default(false) }}
+  evpn_gateway_evpn_l3_inter_domain: {{ switch_data.combined.evpn_gateway.evpn_l3.inter_domain | arista.avd.default(evpn_ebgp_gateway_inter_domain) }}
+  evpn_gateway_vpn_ipv4: {{ switch_data.combined.evpn_gateway.vpn_ipv4.enabled | arista.avd.default(false) }}
+{%     if switch_evpn_gateway is arista.avd.defined(true) %}
+  evpn_gateway_vxlan_route_servers: {{ switch_data.combined.evpn_gateway.remote_peers | arista.avd.default() }}
```

Comment:
why rename from `remote_peers` to `route_servers` ? It makes it harder to read/understand the code in the other templates. I would suggest `evpn_gateway_remote_peers`

Reply:
+1 on evpn_gateway_remote_peers.